ssz/src/main/java/tech/pegasys/artemis/ssz/backing/view/ViewUtils.java

```diff
@@ -11,38 +11,37 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package tech.pegasys.artemis.util.backing.view;
+package tech.pegasys.artemis.ssz.backing.view;
 
 import org.apache.tuweni.bytes.Bytes;
 import org.apache.tuweni.bytes.MutableBytes;
-import tech.pegasys.artemis.util.SSZTypes.Bitlist;
-import tech.pegasys.artemis.util.SSZTypes.Bitvector;
-import tech.pegasys.artemis.util.backing.ListViewRead;
-import tech.pegasys.artemis.util.backing.ListViewWrite;
-import tech.pegasys.artemis.util.backing.VectorViewRead;
-import tech.pegasys.artemis.util.backing.VectorViewWrite;
-import tech.pegasys.artemis.util.backing.type.BasicViewTypes;
-import tech.pegasys.artemis.util.backing.type.ListViewType;
-import tech.pegasys.artemis.util.backing.type.VectorViewType;
-import tech.pegasys.artemis.util.backing.view.BasicViews.BitView;
-import tech.pegasys.artemis.util.backing.view.BasicViews.ByteView;
+import tech.pegasys.artemis.ssz.SSZTypes.Bitlist;
+import tech.pegasys.artemis.ssz.SSZTypes.Bitvector;
+import tech.pegasys.artemis.ssz.backing.ListViewRead;
+import tech.pegasys.artemis.ssz.backing.ListViewWrite;
+import tech.pegasys.artemis.ssz.backing.VectorViewRead;
+import tech.pegasys.artemis.ssz.backing.VectorViewWrite;
+import tech.pegasys.artemis.ssz.backing.type.BasicViewTypes;
+import tech.pegasys.artemis.ssz.backing.type.ListViewType;
+import tech.pegasys.artemis.ssz.backing.type.VectorViewType;
 
 /** Handy view tool methods */
 public class ViewUtils {
 
   /** Creates immutable vector of bytes with size `bytes.size()` from {@link Bytes} value */
-  public static VectorViewRead<ByteView> createVectorFromBytes(Bytes bytes) {
-    VectorViewType<ByteView> type = new VectorViewType<>(BasicViewTypes.BYTE_TYPE, bytes.size());
+  public static VectorViewRead<BasicViews.ByteView> createVectorFromBytes(Bytes bytes) {
+    VectorViewType<BasicViews.ByteView> type =
+        new VectorViewType<>(BasicViewTypes.BYTE_TYPE, bytes.size());
     // TODO optimize
-    VectorViewWrite<ByteView> ret = type.getDefault().createWritableCopy();
+    VectorViewWrite<BasicViews.ByteView> ret = type.getDefault().createWritableCopy();
```

Comment:
nit: Direct import for ByteView and BitView.

Reply:
Done.